README.md

```diff
@@ -1,37 +1,116 @@
+- Install [NodeJS](https://nodejs.org/en).
+- Install or enable [Yarn](https://yarnpkg.com/getting-started/install).
+- Clone this repository.
+- Run `yarn install` in the cloned repository to download all dependencies.
+- Run `yarn start` to start a local development server.
+- Optionally set up [KWoC Backend](https://github.com/kossiitkgp/kwoc-backend) locally and set the `BACKEND_URL` in the [constants.js](./src/constants.js) file to the local URL for the backend.
 
-You can find further instructions in [Learn](learn.md) Page.
-
-# Setting up Locally
-
-## Install Instructions
+<p align="right">(<a href="#top">back to top</a>)</p>
 
+## Project Structure
+### File Structure
 ```
-yarn
+.
+├── public
+└── src
+   ├── assets
+   ├── components
+   ├── data
+   ├── hooks
+   ├── styles
+   ├── utils
+   ├── views
+   ├── constants.js
+   └── App.js
 ```
+- `public`: Contains public files such as `index.html`.
+- `src`: Contains source files (JS, SCSS, assets, etc.)
+  - `assets`: Contains assets used in the source, such as images and icons.
+  - `components`: Contains reusable react components.
+  - `data`: Contains raw data.
+  - `styles`: Contains all stylesheets (CSS, SCSS).
+  - `utils`: Contains commonly used util functions.
+  - `views`: Contains views for each of the pages.
+  - `constants.js`: Contains globally used constants.
+  - `App.js`: Contains the top-level `App` component.
 
-## Running on Local host
+### Libraries/Frameworks Used
+- [React](https://reactjs.dev): For redefining the truth.
+- [SASS](https://sass-lang.com/): For redefining CSS styles.
+- [Feather Icons](https://feathericons.com/) and [Fontawesome Icons](https://fontawesome.com/).
+- And [many more](./package.json).
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-```
-yarn start
-```
+## Archival
+See also [KWoC Docs](https://github.com/kossiitkgp/docs/blob/master/events/kwoc.md#certificates).
 
-## Create Build files
+After KWoC ends, the website is archived. The stats data (including the total PRs, total commits, total contributors, and individual stats) is copied and added to the site source.
 
-```
-yarn build
-```
+The archived source code is committed to a branch named `kwoc-xx-archive` and hosted on `kwocxx.kossiitkgp.org`, where `xx` represents the last two digits of the year. The archive is also hosted on `kwoc.kossiitkgp.org` until the following year's website is deployed.
+
+The development for the next KWoC website continues in the default branch.
+
+> **WARNING** DO NOT COMMIT ANY PERSONAL DATA TO THE ARCHIVE.
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGMENTS -->
+## Acknowledgments
+* [Choose an Open Source License](https://choosealicense.com)
+* [Img Shields](https://shields.io)
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+> Please update this documentation if you make changes to the KWoC frontend or any other part of KWoC which affects the frontend. Future humans will praise you.
```

Comment:
Keep it generic don't mention about frontend specifically, just to stay in sync with the template we will be making. Uniformity is the first step in decluttering.

Reply:
👀 
Suggest a new sentence please, sir.

Reply:
If you don't have a suggestion, I believe we can resolve this :slightly_smiling_face:

Reply:
Also we can make this sentence the template, just replacing KWoC Frontend with whichever project

Reply:
repository -> project

Reply:
shifted to another review section. resolving this.